courses/fundamentals_of_ada/adv_240_tasking.rst

```diff
@@ -226,6 +226,61 @@ Example: Main
 Quiz
 ------
 
+.. code:: Ada
+
+    task type T is
+        entry Go;
+    end T;
+
+    task body T is
+    begin
+        accept Go do
+            loop
+                null;
+            end loop;
+        end Go;
+    end T;
+
+    My_Task : T;
+
+What is the result of calling :ada:`My_Task.Go` from "Main"?
+
+A. Compilation error
+B. Runtime error
+C. :answer:`Main hangs`
+D. :answer:`My_Task hangs`
+
+------
+Quiz
+------
+
+.. code:: Ada
+
+    task type T is
+        entry Go;
+    end T;
+
+    task body T is
+    begin
+        accept Go;
+        loop
+            null;
+        end loop;
+    end T;
+
+    My_Task : T;
+
+What is the result of calling :ada:`My_Task.Go` from "Main"?
+
+A. Compilation error
+B. Runtime error
+C. Main hangs
```

Comment:
This (and the previous) are confusing because we the previous slide showed a "main" without a call to the task.
In addition, in the previous quiz "main" will hang at the rendezvous (as you implied in the answer) but in this example, if  "main" is the main application, then the main application will still hang, because it cannot complete until all tasks are terminated